src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -333,6 +347,37 @@ suggestSignature isQuickFix Diagnostic{_range=_range@Range{..},..}
               | otherwise                  = nm <> ty
 suggestSignature _ _ = []
 
+suggestNewImport :: ExternalPackageState -> ParsedModule -> Diagnostic -> [(T.Text, [TextEdit])]
+suggestNewImport eps ParsedModule {pm_parsed_source = L _ HsModule {..}} Diagnostic{_message}
+  | Just [name, _typ] <- matchRegex (unifySpaces _message) "Variable not in scope: ([^ ]+) :: ([^*•]+)"
```

Comment:
Just to point it out, this doesn't suggest qualified imports. E.g.
```
foo = Maybe.fromJust
```
Produces the error message
```
Not in scope: ‘Maybe.fromJust’
No module named ‘Maybe’ is imported.
```

Supporting qualified imports could be a nice extension of this feature in the future.

Reply:
Agreed! I suspect that there are many extensions to this feature, and I wouldn't be able to tackle them all in this PR.